Userland/Libraries/LibWeb/Platform/AudioCodecPluginGeneric.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibAudio/PlaybackStream.h>
+#include <LibWeb/Platform/AudioCodecPlugin.h>
+
+namespace Web::Platform {
+
+class AudioCodecPluginGeneric final : public Web::Platform::AudioCodecPlugin {
```

Comment:
hmm I'm not convinced on this name. 

But, meh? 

I feel like this should be the "default" LibWeb implementation with the `AudioCodecPlugin` name, and we should call the interface class something else. AbstractAudioCodecPlugin?

We need a lot of polish and such regarding these platform hooks though so it's fine to keep this like this, just food for thought.

Reply:
I'll call it `AudioCodecPluginAgnostic` instead if that helps it to be clearer. It's worth noting, though, the `AudioCodecPlugin` class itself shouldn't need to exist at all once we have enough implementations available in LibAudio.